README.md

```diff
@@ -26,7 +26,12 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+def offerRose(person)
+  puts "Would you take this rose and help out
+  an old beggar," person "?"
+end
+
+offerRose(YoungPrince)
 ```
```

Comment:
You've got the structure, but the syntax is off in a couple of places and would error out. First, you need to use either string concatenation of string interpolation to assemble your `puts